src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -426,7 +426,6 @@ export class GwtCallback extends EventEmitter {
       const rInstalls = findRInstallationsWin32();
       if (rInstalls.length === 0) {
         logger().logErrorMessage('No R installations found via registry or common R install locations.');
```

Comment:
```suggestion
        logger().logInfoMessage('No R installations found via registry or common R install locations.');
```

Given that this isn't really an error condition?